scripts/getRecentUsers.js

```diff
@@ -0,0 +1,46 @@
+require('dotenv').load();
+
+let request = require('request');
+let fs = require('fs');
+
+
+module.exports.run = function run() {
+    
+    let date = new Date();
+    date.setDate( date.getDate() - 1 ); // Remove one day
+    const requestUrl =
+        process.env.API_URL + '/data/game'+
+            '?filter=endTime=gt="'+date.toISOString()+'"'+
+            '&include=playerStats.player'+
+            '&fields[player]=login'+
+            '&sort=-endTime'
+        ;
+     
+	request(
+        requestUrl,        
+        function (error, response, body) {
+            if (! error) {
+                let entries = JSON.parse(body);
+                let finalArray = [];
+
+                if (entries.included == undefined) return;
+
+                for(let i = 0; i < entries.included.length; i++) {
+                    let entry = entries.included[i];
+                    if (entry.type != "player") continue;
+
+                    finalArray.push(entry.attribute.login);
+                }
+
+                fs.writeFile("members/recent.json", JSON.stringify(finalArray), function(error) {
```

Comment:
Do you always have right to write on filesystem? Is website running as root or directory owner?

Reply:
Yes it is, that's how caching is done for the leagues and leaderboards, so I'm merely copying what they do so we don't have 3 different caching systems for 3 systems